### PR TITLE
Replaced loading messages with material ui circularProgress spinners

### PR DESCRIPTION
**Pull request by bparazin**
image

Taking screenshots is a little tricky since they load really pretty fast

Addresses #2343

### CODE REVIEW

**Comment by acrellin** [comment]
@bparazin Looks like a good start, thanks! I see the following additional files all have loading messages that need to be replaced with spinners:
static/js/components/AirmassPlot.jsx
static/js/components/AssignmentForm.jsx
static/js/components/FollowupRequestForm.jsx
static/js/components/Groups.jsx
static/js/components/ManageDataForm.jsx
static/js/components/NonMemberGroupList.jsx
static/js/components/Observability.jsx
static/js/components/PhotometryTable.jsx
static/js/components/RecentSources.jsx
static/js/components/RunSummary.jsx
static/js/components/SourceDesktop.jsx
static/js/components/SourceMobile.jsx
static/js/components/SourceQuickView.jsx
static/js/components/SourceTable.jsx
static/js/components/UploadPhotometry.jsx
static/js/components/UploadSpectrum.jsx
static/js/components/UserInfo.jsx

**Comment by bparazin** [comment]
Ah thank you! I only checked for <b>Loading... </b>, so I missed those! On it in 1 sec

**Comment by bparazin** [comment]
Ok @acrellin that should be all of them?

**Comment by bparazin** [comment]
Ack, let me fix this test

**Comment by bparazin** [comment]
??? The test is passing every time on my machine ???

**Comment by bparazin** [comment]
image

File: static/js/components/ManageDataForm.jsx [hunk 1]
```diff
@@ -148,7 +149,14 @@ const SpectrumRow = ({ rowData, route }) => {
       <TableCell colSpan={colSpan}>
         <Grid container direction="row" justify="center" alignItems="center">
           <Grid item className={styles.photometryContainer} sm={6}>
-            <Suspense fallback={<div>Loading spectroscopy plot...</div>}>
+            <Suspense
+              fallback={
+                <div>
+                  {" "}
```
Comment (acrellin) [inline static/js/components/ManageDataForm.jsx:155]:
```suggestion
```

**Review by acrellin — CHANGES_REQUESTED** [review]
Looks good! Just one request below -- removing unnecessary whitespace. Thanks!

File: static/js/components/ManageDataForm.jsx (hunk 1 above)
Comment (bparazin) [reply static/js/components/ManageDataForm.jsx]: Fixed

**Review by acrellin — APPROVED** [review]

**Comment by acrellin** [comment]
Will merge when green. Thanks @bparazin !

**Comment by acrellin** [comment]
Sole test failure is unrelated.

**Pull request merged**